content/03-reference/01-tools-and-interfaces/01-prisma-schema/index.mdx

```diff
@@ -150,40 +156,53 @@ DATABASE_URL=postgresql://test:test@localhost:5432/test?schema=public
 
 When running any command that needs to access the database defined via the `datasource` block (e.g. `prisma introspect`), the Prisma CLI automatically loads the `DATABASE_URL` environment variables from the `.env` file and makes it available to the CLI.
```

Comment:
Line 150 above "This requires ..." is not really correct. The `DATABASE_URL` can also be supplied from the machine itself - the .env file is just another variation. Not sure if this should be mentioned here.